Compression/fcl/prolog.fcl

```diff
@@ -0,0 +1,204 @@
+BEGIN_PROLOG
+
```

Comment:
This prolog creates circular dependencies with JobConfig, in that JobConfig must source this prolog, but this prolog uses names set by JobConfig.  I know this is how we did this for MDC2018, but I now prefer a one-directional config structure.  I suggest moving most of this content into the appropriate JobConfig prolog (primary or pileup).

Reply:
I've reoriganised the prolog in commit 57645ffe4